bin/ebuild-helpers/prepallstrip

```diff
@@ -1,11 +1,8 @@
 #!/bin/bash
```

Comment:
You may want to keep `prepallstrip` and `prepstrip` working. Some ebuilds are still using them (with `RESTRICT=strip`), and they will no longer be stripped in old EAPIs.